acceptance/launcher_test.go

```diff
@@ -224,14 +176,12 @@ func buildLauncher(t *testing.T) {
 	cmd.Env = append(
 		os.Environ(),
 		"PWD="+cmd.Dir,
+		"BUILD_DIR="+launcherBinaryDir, // `make clean` should remove the launcher binary dir, not the default dir (out)
```

Comment:
I think removing the `launcherBinaryDir` with `make clean` is what we want and also potentially fixes a race condition with the analyzer tests relying on `out`. But maybe as mentioned above, that whole strategy needs re-working...

Reply:
good catch! However, if the `analyzer` test are going to use the binaries from the default `out` dir I do think we should change the `launcher` tests to be consistent with that.

Reply:
Making this change, but then does it make sense to remove the `make clean` part? Otherwise, running the launcher tests will cause the detector/analyzer/etc. tests to all run slower.